src/app/core/basic-datatypes/entity/entity-block/entity-block.component.html

```diff
@@ -0,0 +1,27 @@
+<span
```

Comment:
actually renamed from display-entity.component and turned into a simpler component here